runtime/jit_vm/ctsupport.cpp

```diff
@@ -202,6 +202,8 @@ jitParseSignature (const J9UTF8 *signature, U_8 *paramBuffer, UDATA *paramElemen
 			state = returnValue;
 		} else {
 			switch (*sigChar) {
+// VALHALLA_TODO:  Need to return a J9_NATIVE_TYPE_VALUE for 'Q' in future
+			case 'Q': /* FALLTHROUGH */
```

Comment:
Minor nitpick:  We usually avoid `//` comments in vm code even though they'll compile on all platforms in c++
```suggestion
			case 'Q': 
				/* VALHALLA_TODO:  Need to return a J9_NATIVE_TYPE_VALUE for 'Q' in future */
				/* FALLTHROUGH */
```

Reply:
> Minor nitpick: We usually avoid `//` comments in vm code even though they'll compile on all platforms in c++

Sorry!  I should have read the coding standards before submitting these changes.